dev-lang/mono/mono-4.4.1.0.ebuild

```diff
@@ -50,6 +45,11 @@ pkg_setup() {
 	mono-env_pkg_setup
 }
 
+src_unpack() {
+	default
+	mv "${WORKDIR}/${PN}-$(get_version_component_range 1-3)" "${S}"
```

Comment:
in my expirience, changing default S adds additional troubles with multilib. But if this change is unlikely, I can revert it...